browser/BUILD.gn

```diff
@@ -289,6 +291,17 @@ source_set("browser_process") {
     ]
   }
 
+  if (unstoppable_domains_enabled) {
+    sources += [
+      "unstoppable_domains/unstoppable_domains_service_delegate_impl.cc",
```

Comment:
this should go in a new BUILD.gn file inside unstoppable_domains

Reply:
you're also missing deps here. The problem with putting all this stuff in the same file (other than readability) is that it becomes very difficult to track deps and when they need to be added/removed

Reply:
It was put here because of the need of g_brave_browser_process.
I think the only two deps I needed, `//brave/components/unstoppable_domains` and `//components/keyed_service/content` are already in or added to this target.

Moving brave_browser_process_impl to its own target needs to fix some files it included that are currently belong to browser target, after working on it for a while, I think the scope is too big to include in this PR and better do it in a separate PR.

Reply:
Added a separate target in 5446cc6d69c1e00b9c7c371cf294a760d9e43e5d.
https://github.com/brave/brave-browser/issues/14441 is opened to track the removing of check_includes=false.